keyboards/mechmini/v2/keymaps/wsturgiss/config.h

```diff
@@ -0,0 +1,86 @@
+/* Copyright 2018 TurboMech /u/TurboMech <discord> @A9entOran9e#6134
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ *This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#ifndef V2_CONFIG_H
```

Comment:
You don't need all of this.  Most of this could be removed, actually. 

In fact, this entire file could be slimmed down to just this:
```c
#pragma once

#define TAPPING_TERM 180 

 /* space cadet stuff */
#define LSPO_KEY KC_9
#define RSPC_KEY KC_0
#define DISABLE_SPACE_CADET_ROLLOVER
```

Reply:
Done.